src/sidebar/components/group-list.js

```diff
@@ -75,6 +75,10 @@ function GroupListController($window, analytics, groups, settings, serviceUrl) {
   }
 
   this.isThirdPartyService = isThirdPartyService(settings);
+
+  this.showGroupsMenu = function() {
```

Comment:
For new code I would recommend that we use arrow functions rather than anonymous functions. That makes the behavior of `this` easier to reason about - it will always be whatever `this` was in the scope at the time of the declaration instead of depending on how the function is called.

At some point I plan to do a bulk update of existing code using automated tools.